components/participant_table/commons/participant_table_base.lua

```diff
@@ -157,20 +157,17 @@ function ParticipantTable:readSection(args)
 	local config = self.readConfig(args, self.config)
 	local section = {config = config}
 
-	local keys = Array.filter(Array.extractKeys(args), function(key)
-		return String.contains(key, '^%d+$') or
-			String.contains(key, '^p%d+$') or
-			String.contains(key, '^player%d+$')
-	end)
-
 	local entriesByName = {}
```

Comment:
this drops support for unnamed (indexed) arguments, is that by choice?

Reply:
does not
`mapArgumentsByPrefix` interleaves with unnamed args

Reply:
Ah right, i always forget about that feature of the base `mapArguments`